Userland/Applications/Browser/Tab.cpp

```diff
@@ -257,7 +257,7 @@ Tab::Tab(BrowserWindow& window)
     m_link_context_menu->add_separator();
     m_link_context_menu->add_action(window.inspect_dom_node_action());
 
-    view().on_link_context_menu_request = [this](auto& url, auto& screen_position) {
+    view().on_link_context_menu_request = [this](auto& url, auto screen_position) {
```

Comment:
Not something you have to do, just a general thought:
Maybe spelling the types here would help, or make it more clear what is exactly happening with these lambdas

Reply:
AFAIK it is the recommended coding style in SerenityOS to use auto where possible.